lib/datastore_api/requests/update_application.rb

```diff
@@ -40,7 +40,7 @@ def call
 
       def endpoint
         format(
-          '/applications/%<resource_path>s', resource_path: resource_path
+          '/applications/%<resource_path>s', resource_path:
```

Comment:
Still have to double take sometimes when I see this new syntax but it is cleaner, and you have to keep up with the times 😆

Reply:
Yeah, sometimes feels kind of pointless or even confusing, but it seems like it has the nice effect of keeping variable naming consistent and also can save quite a big chunk of boilerplate code depending sometimes for big hash structures.

Reply:
Yep agree its a nice addition :)